api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -57,17 +57,29 @@ DbUser createUser(
 
   void validateAllOfUsTermsOfService(Integer tosVersion);
 
-  boolean validateAllOfUsTermsOfServiceVersion(@Nonnull DbUser dbUser);
-
-  boolean validateTermsOfService(@Nonnull DbUser dbUser);
+  /**
+   * Is the user up-to-date with our Terms of Service?
+   *
+   * @param dbUser the current user - used only to generate error messages
+   * @return true only if the user has accepted the latest version of the AoU Terms of Service
+   */
+  boolean hasSignedLatestAoUTermsOfService(@Nonnull DbUser dbUser);
 
-  boolean getUserTerraTermsOfServiceStatus(@Nonnull DbUser dbUser);
```

Comment:
Do we know why getUserTerraTermsOfServiceStatus was declared a method initially? Was it being used in login/create account flow